frontend/src/scenes/teamLogic.ts

```diff
@@ -136,6 +141,12 @@ export const teamLogic = kea<teamLogicType>({
             if (contextualTeam) {
                 // If app context is available (it should be practically always) we can immediately know currentTeam
                 actions.loadCurrentTeamSuccess(contextualTeam)
+                if (contextualTeam.ingested_event) {
+                    const builtEventUsageLogic = eventUsageLogic.build()
+                    const unmount = eventUsageLogic.mount()
```

Comment:
Hmmm, this event is going to be pretty noisy. Is the reason we don't fire it from the ingestion screen that a user might select "skip" and then still ingest events later?

If we don't have a better way of doing this, can we name the event a bit better, so future us don't do analysis based on counts around this event. And maybe we should comment this too.

Reply:
Also, it looks like the reason we're doing the funky mounts here is because of a cyclical connection between preflightLogic->eventUsageLogic->teamLogic->preflightLogic. We should probably add that comment too, so future us know what's going on.

Reply:
This event exists because we don't have a way to measure "team has event ingested" in experiments right now since we don't have support for other libraries. Under @neilkakkar's suggestion here https://posthog.slack.com/archives/C03DDAFRA9L/p1651853504598879?thread_ts=1651852421.656819&cid=C03DDAFRA9L